docs/geedocs/answer/7160000.html

```diff
@@ -148,10 +149,95 @@
           <td>Close the resource form and open it again to make the save option available again.</td>
         </tr>
         <tr>
+          <td>340</td>
+          <td>GE Fusion Terrain is black</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
           <td>342</td>
           <td>Fusion crashes when opening an unsupported file type</td>
           <td>Re-open fusion and avoid opening unsupported file types.</td>
         </tr>
+        <tr>
+          <td>375</td>
+          <td>unbundle psycopg2</td>
+          <td>On Ubuntu 16.04 switch to python-psycopg2' instead of 'python2.7-psycopg2'.</td>
+        </tr>
+        <tr>
+          <td>380</td>
+          <td>Provider field in resource-view is blank</td>
+          <td>No current work around</td>
+        </tr>
+        <tr>
+          <td>401</td>
+          <td>GEE commands are not in the path for sudo. </td>
+          <td>Add /opt/google/bin to the path for all users, including the super user</td>
```

Comment:
Minor: another workaround is to specify the full path when running commands.  Not sure if it's worth adding that that.

Reply:
Sure. I'll add it.